### ISSUE TITLE
iOS build issue Xcode 10.3

### ISSUE THREAD

**Issue by jakupi1** [dpa99c/cordova-plugin-firebasex#60]
Hello im unable to build a new version of app when i try to use ionic cordova build ios 
this error shows 
```
Executing script found in plugin cordova-plugin-firebasex for hook "after_prepare": plugins/cordova-plugin-firebasex/scripts/after_prepare.js
Preparing Firebase on iOS
No scripts found for hook "before_compile".
Cannot read property 'name' of undefined
TypeError: Cannot read property 'name' of undefined

```

**Comment by dpa99c** [comment]
I would guess that's raised by [this line](https://github.com/dpa99c/cordova-plugin-firebasex/blob/master/scripts/after_prepare.js#L15) and due to `<name>` missing from `config.xml`.

Please build and run the [example project](https://github.com/dpa99c/cordova-plugin-firebasex) then use it as a working reference for your own project.

**Comment by jakupi1** [comment]
There is name in config.xml, and also i was able to build and yesterday i couldn't build anymore there was error with the plugin you removed the `git+https://github.com/dpa99c/cordova-plugin-firebase.git#GH-1057-April-05-android-build-issue` 
`<name>Ionic App</name>`

**Comment by dpa99c** [comment]
Please try to clone and build [example project](https://github.com/dpa99c/cordova-plugin-firebasex) for a known comparison.

**Comment by jakupi1** [comment]
I followed this steps and still cant build still getting the same error
`Preparing Firebase on iOS
Cannot read property 'name' of undefined`
`git clone https://github.com/dpa99c/cordova-plugin-firebasex-test.git && cd cordova-plugin-firebasex-test
npm install

cordova platform add ios
cordova run ios`

**Comment by dpa99c** [comment]
This suggests a problem with your dev environment - the same steps work fine in my env (see [console log output](http://ge.tt/2ghitGx2)).

One idea: check there are no non-latin characters in the directories of the absolute path to your project - these have been known to cause issues.

**Comment by jakupi1** [comment]
Here it is my ionic info, we are tying at another Mac the error is the same maybe the problem is from versions?
```
Ionic:

   Ionic CLI                     : 5.2.3 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.5.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.2.4
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.4.1

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.1.0, (and 18 other plugins)

Utility:

   cordova-res : 0.5.0 (update available: 0.6.0)
   native-run  : 0.2.2 (update available: 0.2.8)

System:

   Android SDK Tools : 26.1.1 (/Users/mjakupi/Library/Android/sdk)
   ios-deploy        : 1.9.4
   ios-sim           : 8.0.1
   NodeJS            : v10.16.0 (/usr/local/Cellar/node@10/10.16.0/bin/node)
   npm               : 6.9.0
   OS                : macOS Mojave
   Xcode             : Xcode 10.3 Build version 10G8
```

**Comment by jakupi1** [comment]
I updated Xcode and it fixed the error!
@dpa99c  thank you a lot !
